src/content/docs/en/recipes/bun.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Use Bun with Astro
+description: Learn how to use Bun with your Astro site.
+type: recipe
+i18nReady: false
+---
+
+[Bun](<https://bun.sh/>) is a newer runtime for JavaScript built for speed. See [Bun's documentation](<https://bun.sh/docs>) for more information.
+
+Using Bun with Astro still is experimental, not all integrations might work as expected. If you find any issues, please open an issue!
+
+## Installing Bun
+
+Installing Bun is a breeze - just run one simple command in your terminal and you're ready to go! Plus, with support for `npx packages, creating a new Astro project with Bun is as easy as pie.
+
+Here's how you can install Bun:
+
+```bash
+curl -fsSL https://bun.sh/install | bash
+```
+
+## Create a new Astro project with Bun
+
+Bun has support for `npx` packages, which means you can create a new Astro project with Bun in one familiar command:
+
+```bash
+bunx create-astro@latest my-astro-project-using-bun
+```
+
+:::tip[Tip]
+If you want to use a theme, that will work as well!
+```bash
+bunx create-astro@latest my-astro-project-using-bun --template eliancodes/brutal
+```
+:::
+
+## Install your dependencies
+
+If you're starting a new project using `bunx create-astro`, you can skip this step, the CLI will automatically use Bun to install dependencies. Otherwise, you'll need to install your dependencies with Bun:
+
+```bash
+bun install
+```
+
+## Running the development server
+
+Running the development server with Bun should feel quite familiar:
+
+```bash
+bun run dev
+```
+
+:::note[Note]
+Keep in mind that using `bun dev` will not work, as that will start Bun's internal dev server and not your Astro development server.
+:::
+
+## Using Astro integrations
+
+But the best part? Bun makes it simple to integrate with all of your favorite Astro tools. With support for npx packages, you can use any of the Astro integrations with Bun in one familiar command:
+
+```bash
+bunx astro add react
+```
+
+This will work exactly the same as if you were using NPM, but with the added benefit of using Bun's blazing fast runtime.
+
+:::note[Note]
+1. Some integrations might not work as expected. If you find any issues, please open an issue!
+2. Keep in mind that using `bun add` will not work, as that will add the package to Bun's dependencies and not your Astro project.
+:::
```

Comment:
```suggestion
:::note[Note]
Some integrations might not work as expected. If you find any issues, please open an issue on GitHub!
:::
```
Also reminder to link to the appropriate GitHub repo here.

I removed the line about `bun add` because I think it's reasonable to expect people to follow the command as given here. The one about running `dev` is reasonable because a lot of people do use the shorter command, maybe even without thinking about it. And, you're running that command so often compared to adding an integration.